decidim/decidim
# Fix cryptic file validation errors

#### :tophat: What? Why?
Currently the file validation messages are rather cryptic for the general user because they are showing the content type matching pattern regular expressions converted into strings.

To make it understandable for the general public, we want to show the file extensions instead of the content types.

#### :pushpin: Related Issues
- Fixes #9185

#### Testing
- Enable attachments for proposals
- Try to upload an attachment which is not allowed, e.g. a `.zip` file is not allowed by default
- See a better error message
